From c987007886c4b4cfff31f823fbeee50145c3e587 Mon Sep 17 00:00:00 2001
From: "tong.gao" <tong.gao@amlogic.com>
Date: Fri, 14 Jul 2023 10:03:02 +0800
Subject: [PATCH 2/3] camctrl: delete pixformat [1/1]

PD#SWPL-112855

Problem:
rgb is blue

Solution:
use format from gst

Verify:
local

Signed-off-by: tong.gao <tong.gao@amlogic.com>
Change-Id: I9677b937060fd70e86425213b222fbd0c33def5f
---
 camctrl/camctrl.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/camctrl/camctrl.cc b/camctrl/camctrl.cc
index b5160fd..6623228 100644
--- a/camctrl/camctrl.cc
+++ b/camctrl/camctrl.cc
@@ -109,7 +109,7 @@ struct thread_param {
   /* format related */
   uint32_t width;
   uint32_t height;
-  uint32_t pixformat;
+  // uint32_t pixformat;
   uint32_t fmt_code;
   uint32_t wdr_mode;
 
@@ -132,7 +132,7 @@ struct thread_param {
 static struct thread_param tparam = {
   .width = 3840,
   .height = 1920,
-  .pixformat = V4L2_PIX_FMT_NV12, /* V4L2_PIX_FMT_NV12, V4L2_PIX_FMT_Y12, V4L2_PIX_FMT_SRGGB12 */
+  // .pixformat = V4L2_PIX_FMT_NV12, /* V4L2_PIX_FMT_NV12, V4L2_PIX_FMT_Y12, V4L2_PIX_FMT_SRGGB12 */
 #ifdef WDR_ENABLE
   .fmt_code = MEDIA_BUS_FMT_SBGGR10_1X10, /* MEDIA_BUS_FMT_SRGGB12_1X12 */
   .wdr_mode = WDR_MODE_2To1_FRAME, /* WDR_MODE_2To1_LINE */
@@ -382,7 +382,7 @@ int media_stream_config(media_stream_t *stream) {
   memset(&stream_cfg, 0, sizeof(stream_configuration_t));
   stream_cfg.format.width = tparam.width;
   stream_cfg.format.height = tparam.height;
-  stream_cfg.format.fourcc = tparam.pixformat;
+  // stream_cfg.format.fourcc = tparam.pixformat;
   stream_cfg.format.code = tparam.fmt_code;;
   stream_cfg.format.nplanes = 1;
 
-- 
2.25.1

